docker/run_celery.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+while ! nc -z rabbit 5672; do echo "Waiting for RabbitMQ... \n"; sleep 3; done
```

Comment:
I added this because the standard behaviour was waiting on port 5673 (I think it was) and that did not work.